lib/include/jxl/encode.h

```diff
@@ -283,6 +283,12 @@ typedef enum {
    */
   JXL_ENC_FRAME_INDEX_BOX = 31,
 
+  /** Sets brotli encode effort for use in JPEG recompression and compressed
+   * metadata boxes (brob). Ranges from 0 (fastest) to 11 (slowest).
```

Comment:
LGTM, but it would be better to have looser coupling with brotli encoder.
Imagine, one day we have brotli 2.0, which have, e.g. quality 12.
For JXL users 11 meant max compression and it will no longer be true.
So, would be cool to have some coarse grained quality setting here (e.g. 1..5) and choose brotli quality inside codec.

Reply:
I see what you mean, but if and when that happens, can't we just remap 0-11 to whatever the new range would be?

I don't see why it would help if the effort setting is more coarse-grained. If anything, I would make it more fine-grained, say a setting from 0 to 100 that maps to `setting/9`, leaving room for future brotli changes.

I like the 0..11 scale for effort though, and I hope brotli will keep it, and libjxl will eventually expand its current 1..9 effort scale to 0..11 too :)